<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Acountant</class>
 <widget class="QMainWindow" name="Acountant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ხარჯთაღმრიცხველი</string>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="client_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>159</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>კლიენტის სახელი</string>
    </property>
   </widget>
   <widget class="QLabel" name="deadline_visit_label">
    <property name="geometry">
     <rect>
      <x>-7</x>
      <y>50</y>
      <width>271</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>დედლაინით განსაზღვრული      ვიზიტის დრო</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="actual_visit_label">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>60</y>
      <width>204</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ფაქტობრივი ვიზიტის დრო</string>
    </property>
   </widget>
   <widget class="QLabel" name="condition_label">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>67</y>
      <width>186</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ობიექტის მდგომარეობა</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="measurement_label">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>180</y>
      <width>124</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>კვადრატულობა</string>
    </property>
   </widget>
   <widget class="QLabel" name="deadline_label">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>150</y>
      <width>237</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>დედლაინით განსაზღვრული  ხარჯთაღრიცხვის დასრულების
 დრო</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="actual_label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>160</y>
      <width>199</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ფაქტობრივი დასრულების
                თარიღი</string>
    </property>
   </widget>
   <widget class="QLabel" name="unitpriceingel_label">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>284</y>
      <width>175</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ერთეულის ფასი ლარში</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="measurement">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="unitpriceingel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </widget>
   <widget class="QDateEdit" name="deadline_date">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>220</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="actual_date">
    <property name="geometry">
     <rect>
      <x>536</x>
      <y>220</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="deadline_visit_date">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2022</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::DaySection</enum>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="timeSpec">
     <enum>Qt::LocalTime</enum>
    </property>
    <property name="date">
     <date>
      <year>2022</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QDateEdit" name="actual_visit_date">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>110</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="client_list_label">
    <property name="geometry">
     <rect>
      <x>859</x>
      <y>10</y>
      <width>119</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>კლიენტების სია</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>400</width>
      <height>490</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>441</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="object_condition">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="updateinfo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>230</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ინფორმაციის განახლება</string>
    </property>
   </widget>
   <widget class="QLabel" name="client_name_text">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="unitpriceinusd">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>320</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="unitpriceingel_label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>284</y>
      <width>199</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ერთეულის ფასი დოლარში</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="total_price">
    <property name="geometry">
     <rect>
      <x>536</x>
      <y>320</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="unitpriceingel_label_3">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>284</y>
      <width>162</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ჯამური ღირებულება</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>390</y>
      <width>230</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>დახურვა</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>40</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>deadline_visit_date</tabstop>
  <tabstop>actual_visit_date</tabstop>
  <tabstop>object_condition</tabstop>
  <tabstop>measurement</tabstop>
  <tabstop>deadline_date</tabstop>
  <tabstop>actual_date</tabstop>
  <tabstop>unitpriceingel</tabstop>
  <tabstop>unitpriceinusd</tabstop>
  <tabstop>total_price</tabstop>
  <tabstop>updateinfo</tabstop>
  <tabstop>close_button</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
